client/grafana-data-source/src/server.rs

```diff
@@ -110,12 +110,50 @@ impl<T> tokio_executor::TypedExecutor<T> for Executor
 	}
 }
 
+/// An error that may occur during server runtime.
+#[derive(Debug, derive_more::Display, derive_more::From)]
+pub enum RunError {
+	/// Propagated hyper server error.
+	Hyper(hyper::Error),
+	/// Initial bind IO error.
+	Io(async_std::io::Error),
```

Comment:
This is the same as `std::io::Error`. Use that to make CI pass.

```suggestion
	Io(std::io::Error),
```